doc/modules/gaussian_process.rst

```diff
@@ -77,149 +88,12 @@ the API of standard scikit-learn estimators, GaussianProcessRegressor:
   externally for other ways of selecting hyperparameters, e.g., via
   Markov chain Monte Carlo.
 
+.. topic:: Examples
```

Comment:
maybe rename to `gpr_examples`? we might easily conflict with another place defining an `Examples` topic.

Reply:
This is a topic and not the name of the section. I don't think you can use this to reference.